UnityProject/Assets/Scripts/Core/Cooldowns/Cooldown.cs

```diff
@@ -17,3 +17,9 @@ public class Cooldown : ScriptableObject, ICooldown
 	/// </summary>
 	public float DefaultTime => defaultTime;
 }
+
+public class CooldownInstance : ICooldown
```

Comment:
I don't understand this addition

Reply:
Means you could do ``` var newCooldown = new CooldownInstance{defaultTime = 3f}``` wheras with the SO above you'd need to do ```var newCooldown  = Instantiate(Cooldown);  newCooldown .defaultTime = 3f;```

Can remove it if you think its not really needed

Reply:
okay, I think I'm just worried about having multiple ways to do a single thing. I fel it is better to have one precise way to do stuff and make everyone use the same.